src/main/java/org/folio/inventory/MarcHridSetConsumerVerticle.java

```diff
@@ -78,35 +40,8 @@ public void start(Promise<Void> startPromise) {
   }
 
   @Override
-  public void stop(Promise<Void> stopPromise) {
-    CompositeFuture.join(marcBibConsumerWrapper.stop(), marcHoldingsConsumerWrapper.stop())
-      .onComplete(ar -> stopPromise.complete());
-  }
-
-  private int getLoadLimit() {
-    return Integer.parseInt(System.getProperty("inventory.kafka.MarcBibInstanceHridSetConsumer.loadLimit", "5"));
```

Comment:
The `getLoadLimit` method in `KafkaConsumerVerticle` will not work in this case.
Property value used here is `inventory.kafka.MarcBibInstanceHridSetConsumer.loadLimit`. However, the 
`getLoadLimit` in `KafkaConsumerVerticle` will resolve in a different value `inventory.kafka.MarcHridSetConsumer.loadLimit`

Reply:
Yes, for this class it doesn't follow common pattern. Thank you! Fixed